raiden-ts/src/raiden.ts

```diff
@@ -1387,6 +1397,35 @@ export class Raiden {
     this.store.dispatch(withdraw.request(undefined, meta));
     return promise;
   }
+
+  /**
+   * Fetches an ordered list of suggested partners from provided, configured or first found PFS
+   *
+   * @param token - Token address to get partners for
+   * @param options - Request options
+   * @param options.pfs - PFS to use, instead of configured or automatic
+   * @returns Ordered array of suggested partners, with address and scoring values according to PFS
+   */
+  public async suggestPartners(
+    token: string,
+    options: { pfs?: RaidenPFS } = {},
+  ): Promise<SuggestedPartner[]> {
+    assert(Address.is(token), [ErrorCodes.DTA_INVALID_ADDRESS, { token }], this.log.info);
+    const tokenNetwork = this.state.tokens[token];
+    assert(tokenNetwork, ErrorCodes.RDN_UNKNOWN_TOKEN_NETWORK, this.log.info);
```

Comment:
Do you think it is worth to test this error? Should be a fairly easy test case.
It is not about making sure the code is working, but it is more a behavior check that we expect. Since this is an interface function where the client can us feed with anything. So we have the requirement to this function that it needs to validate the token address before doing the request.

Reply:
I don't think this is worth to be tested. We test the happy case, and the same pattern is used elsewhere and tested for the unhappy case, and this is just simple validation.

Reply:
Then I think we do a bad job in testing the expected behavior.